pkg/ddl/backfilling_dispatcher.go

```diff
@@ -328,6 +353,26 @@ func generateNonPartitionPlan(
 	return subTaskMetas, nil
 }
 
+func (dsp *BackfillingDispatcherExt) getWriterMemSize(backfillMeta *BackfillGlobalMeta) (uint64, error) {
+	failpoint.Inject("mockWriterMemSize", func() {
+		failpoint.Return(1*size.GB, nil)
+	})
+	_, writerCnt := expectedIngestWorkerCnt()
+	memTotal, err := memory.MemTotal()
```

Comment:
each task will takes half of total memory, ok for now, let's change it when impl framework resource control

Reply:
fixed by merge master